src/landing/LoginForm.js

```diff
@@ -19,22 +25,8 @@ const LoginForm = () => {
     }
   };
 
-  const validUsername = () => {
-    if (username.length === 0) {
-      return false;
-    }
-    return true;
-  };
-
-  const validPassword = () => {
-    if (password.length < 6) {
-      return false;
-    }
-    return true;
-  };
-
   const buttonDisabledValue = () => {
```

Comment:
Would name this like `isLoginButtonDisabled`

Reply:
updated function names for validations as well as for the
isLoginButtonDisabled and
isRegisterButtonDisabled